### PR TITLE
Feature/dashboard edits

### PR DESCRIPTION
**Pull request by kavakai**
### What (if any) features are you implementing?
- n/a
### What (if anything) did you refactor?
- Refactored dashboard CSS
### Were there any issues that arose?
- Not yet

### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

**Review by gjtrecny — APPROVED** [review]

File: src/App.scss [hunk 1]
```diff
@@ -6,10 +6,13 @@ html {
   font-family: 'Montserrat', sans-serif;
   height: 100%;
   width: 100%;
+  scroll-behavior: smooth;
```
Comment (anewb87) [inline src/App.scss:9]: Like peanut butter.

File: src/Pages/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -16,18 +17,28 @@ const Dashboard = () => {
   const { user } = useContext(UserContext);
 
   return (
-    <section className='dashboard-page'>
-      <h2>Welcome, {user}! 💎 </h2>
-      <Nav />
-      <article data-testid='dashboard-quote' className='dashboard-quote'>
-        { quote }
+    <section className="dashboard-page">
```
Comment (anewb87) [inline src/Pages/Dashboard/Dashboard.js:20]: Classing up that top banner part.

File: src/SASS/_keyframes.scss [hunk 3]
```diff
@@ -85,4 +85,30 @@
               transform: translateZ(0);
       opacity: 1;
     }
-  }
\ No newline at end of file
+  }
+
+  @-webkit-keyframes roll-in-left {
```
Comment (anewb87) [inline src/SASS/_keyframes.scss:90]: organizing

**Review by anewb87 — APPROVED** [review]
The css tinkering continues! This page is basically done, just working on a hamburger menu that we'll throw in here!

**Pull request merged**